app/Http/Livewire/Delegates/Monitor.php

```diff
@@ -27,7 +31,11 @@ final class Monitor extends Component
     public function render(): View
     {
         return view('livewire.delegates.monitor', [
-            'delegates'  => $this->delegates,
+            'delegates'  => collect($this->delegates)
+                ->each(fn ($slot) => $slot->setFavorite(Settings::hasFavoriteDelegate($slot->publicKey())))
+                ->sortBy(fn ($slot) => ! $slot->isFavorite())
```

Comment:
would try the approach for adding a class or otherwise identifier to the favourite entries so you can match it on that for the border, should be easiest 🤔

Reply:
I went with that approve in the end. I stored the favourite against the slot to save doing hundreds of `Session::hasFavoriteDelegate` every time I wanted to know the value, as that read the cookies each time